.github/workflows/detect-bounty-issue.yml

```
@@ -0,0 +1,44 @@
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  detect-bounty:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check for linked bounty issue
+        uses: actions/github-script@v6
```

actionlint (1.7.9)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)